package.json

```diff
@@ -11,6 +11,7 @@
     "husky": "^8.0.0",
     "lint-staged": "^12.4.3",
     "prettier": "^2.8.8",
+    "ts-to-zod": "^3.8.2",
```

Comment:
Are you using this somewhere? Personally I'd prefer we go zod to TS like we do for our current big schemas (see `ChainMetadataSchema`)

Reply:
i used this to generate some of the schemas during developement. I'll probably need it again. I can remove, if you don't think it should be included

Reply:
Yeah let's remove it unless something is actively using it.
you can always `npx` if you want to one-off run a lib's bin

Reply:
Removed